soccer/radio/USB2015Radio.hpp

```diff
@@ -0,0 +1,28 @@
+#include <hidapi.h>
+#include "Radio.hpp"
+
+class USB2015Radio : public Radio {
+public:
+    static const unsigned int RJ_VENDOR_ID = 0x524A;
+    static const unsigned int RJ_BASE_2015 = 0x4253;
+
+    USB2015Radio(const unsigned int VENDOR_ID = RJ_VENDOR_ID, const unsigned int PRODUCT_ID = RJ_BASE_2015);
+    virtual ~USB2015Radio();
+
+    virtual bool isOpen() const override;
+    virtual void send(Packet::RadioTx& packet) override;
+    virtual void receive() override;
+    virtual void switchTeam(bool blueTeam) override;
+
+    std::string getProtocolVersion();
+
+
+private:
+    static volatile bool hidAPIInit = false;
```

Comment:
i added it so the hid_exit call would be thread safe, but I haven't decided if I want to handle hid_init and hid_exit in this class for sure yet.